### PR TITLE
feat(docs): document `naiveBayesClassifier`

### PR DESCRIPTION
**Pull request by pierwill**

### CODE REVIEW

File: stdlib/contrib/RohanSreerama5/naiveBayesClassifier/naiveBayesClassifier.flux [hunk 1]
```diff
@@ -1,11 +1,25 @@
...
-//Please ensure Ruby is installed
+// Package naiveBayesClassifier provides an implementation of
+// a naive Bayes classifier.
+//
+// Currently supports single field classification and binary data sets.
+// Please ensure Ruby is installed.
+//
+// See [README](https://github.com/influxdata/flux/blob/master/stdlib/contrib/RohanSreerama5/naiveBayesClassifier/README.md) for more information.
+//
+// Final working code as of August 11, 2020.
```
Comment (sanderson) [inline stdlib/contrib/RohanSreerama5/naiveBayesClassifier/naiveBayesClassifier.flux:9]:
```suggestion
// Package naiveBayesClassifier provides an implementation of
// a naive Bayes classifier.
//
// Currently supports single field classification and binary data sets.
//
// For information about demonstrating functions in this package, see the
// [package README on GitHub](https://github.com/influxdata/flux/blob/master/stdlib/contrib/RohanSreerama5/naiveBayesClassifier/README.md).
```

File: stdlib/contrib/RohanSreerama5/naiveBayesClassifier/naiveBayesClassifier.flux [hunk 2]
```diff
@@ -1,11 +1,25 @@
...
 import "system"
 
+
+// naiveBayes performs the classification.
+//
+// ## Parameters
+//
+// - myMeasurement: measurement to use as training data.
+// - myField: field to use as training data.
+// - myClass: class to classify against.
```
Comment (sanderson) [inline stdlib/contrib/RohanSreerama5/naiveBayesClassifier/naiveBayesClassifier.flux:22]:
```suggestion
// naiveBayes performs a naive Bayes classification.
//
// ## Parameters
//
// - myMeasurement: Measurement to use as training data.
// - myField: Field to use as training data.
// - myClass: Class to classify against.
// - tables: Input data. Default is piped-forward data (`<-`).
```

**Review by sanderson — CHANGES_REQUESTED** [review]

File: stdlib/contrib/RohanSreerama5/naiveBayesClassifier/naiveBayesClassifier.flux [hunk 3]
```diff
@@ -1,11 +1,24 @@
...
+
+// naiveBayes performs a naive Bayes classification.
+//
+// ## Parameters
+//
+// - myMeasurement: Measurement to use as training data.
+// - myField: Field to use as training data.
+// - myClass: Class to classify against.
+// - tables: Input data. Default is piped-forward data (`<-`).
 naiveBayes = (tables=<-, myClass, myField, myMeasurement) => {
```
Comment (pierwill) [inline stdlib/contrib/RohanSreerama5/naiveBayesClassifier/naiveBayesClassifier.flux:22]: fluxdoc is having trouble handling `tables=<-`. I'll look into it 😈

Comment (sanderson) [reply stdlib/contrib/RohanSreerama5/naiveBayesClassifier/naiveBayesClassifier.flux:22]: @nathaniel, are we omitting `tables` parameters from parameter lists?

File: stdlib/contrib/RohanSreerama5/naiveBayesClassifier/naiveBayesClassifier.flux [hunk 4]
```diff
@@ -1,11 +1,22 @@
-//Final working code as of August 11, 2020
-//Currently supports single field classification and binary data sets 
-//Please ensure Ruby is installed
+// Package naiveBayesClassifier provides an implementation of
+// a naive Bayes classifier.
+//
+// Currently supports single field classification and binary data sets.
+//
+// For information about demonstrating functions in this package, see the
+// [package README on GitHub](https://github.com/influxdata/flux/blob/master/stdlib/contrib/RohanSreerama5/naiveBayesClassifier/README.md).
```
Comment (sanderson) [inline stdlib/contrib/RohanSreerama5/naiveBayesClassifier/naiveBayesClassifier.flux:7]:
```suggestion
// [package README on GitHub](https://github.com/influxdata/flux/blob/master/stdlib/contrib/RohanSreerama5/naiveBayesClassifier/README.md).
//
// introduced: v0.86.0
//
```

File: stdlib/contrib/RohanSreerama5/naiveBayesClassifier/naiveBayesClassifier.flux [hunk 5]
```diff
@@ -1,11 +1,22 @@
...
 
 import "system"
 
+// naiveBayes performs a naive Bayes classification.
+//
+// ## Parameters
+//
+// - myMeasurement: Measurement to use as training data.
+// - myField: Field to use as training data.
+// - myClass: Class to classify against.
```
Comment (sanderson) [inline stdlib/contrib/RohanSreerama5/naiveBayesClassifier/naiveBayesClassifier.flux:19]:
```suggestion
// - myClass: Class to classify against.
// - tables: Input data. Default is piped-forward data (`<-`).
//
// tags: transformations
//
```

**Review by sanderson — CHANGES_REQUESTED** [review]
Two suggestions.

**Review by rockstar — APPROVED** [review]

**Review by sanderson — APPROVED** [review]

**Pull request merged**